src/core/ee/dmac.hpp

```diff
@@ -11,6 +11,7 @@ struct DMA_Channel
     uint32_t quadword_count;
     uint32_t tag_address;
     uint32_t tag_save0, tag_save1;
+	uint32_t scratchpad_address;
```

Comment:
Remove the excess indentation.

Reply:
For some reasaon it replaced that with a tab instead of spaces, I blame VS :P